Week3/homework/3-step3.js

```diff
@@ -11,6 +16,10 @@ console.log('for', repeatStringNumTimesWithFor('abc', 3));
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
   // add your code here
+  if (num <= 0) return "";
+  const repeat = str;
+  while (num > 1)
+    str += repeat, num--;
```

Comment:
there is no need for the `if` statement, because in any case while number is a truthy *(NOT `0`)* iterate through it.
And the next one...